tests/Apcu/ApcuTest.php

```diff
@@ -37,6 +37,30 @@ public function testApcu()
         $this->assertArrayHasKey('cache_list', apcu_cache_info());
     }
 
+    public function testArrayCompatibility() {
+        // Test that arrays are properly written and read
```

Comment:
It would. phpUnit makes it hard to override built in functions, and I was trying to do a minor contribution rather than an overhaul.  Ideally, I'd like it to be in an object that can be broken down better for unit testing, but that's more than just a minor fix.

There's also the issue of Zend refusing to run CLI, requiring a web-based unit test.